frontend/packages/kubevirt-plugin/src/components/form/k8s-resource-select-row.tsx

```diff
@@ -44,17 +46,29 @@ export const K8sResourceSelectRow: React.FC<K8sResourceSelectProps> = ({
     loadedData = loadedData.filter(filter);
   }
 
+  let nameValue;
+  let missingError;
+
+  if (name && !isLoading && !loadError && !loadedData.some((entity) => getName(entity) === name)) {
+    missingError = `Selected ${name} is not available`;
+  } else {
+    nameValue = name;
+  }
+
   return (
     <FormRow
       title={title || model.label}
       fieldId={id}
       isLoading={isLoading}
-      validationMessage={loadError || (validation && validation.message)}
-      validationType={loadError ? ValidationErrorType.Error : validation && validation.type}
+      validationMessage={loadError || missingError || (validation && validation.message)}
```

Comment:
Would this and where `validationType` is set below be good places to use the new null-coalescing operator instead of `||`?

Reply:
I think we want to skip falsy values like `""` error here

Reply:
Ah, ok. Disregard, then. =)